src/include/connection.h

```diff
@@ -42,6 +42,31 @@ struct __wt_process {
 };
 extern WT_PROCESS __wt_process;
 
+typedef enum __wt_bg_compact_cleanup_type {
+    BACKGROUND_CLEANUP_ALL_STAT,
+    BACKGROUND_CLEANUP_STALE_STAT
+} WT_BACKGROUND_COMPACT_CLEANUP_TYPE;
+
+/* WT_BACKGROUND_COMPACT_STAT --
```

Comment:
Format needs to be fixed here, missing one line

Reply:
Fixed.